nixos/modules/services/x11/display-managers/lightdm.nix

```diff
@@ -212,6 +212,10 @@ in
     # lightdm uses the accounts daemon to rember language/window-manager per user
     services.accounts-daemon.enable = true;
 
+    # Enable the accounts daemon to find lightdm's dbus interface
+    environment.pathsToLink = [ "/share/accountsservice" ];
```

Comment:
I think this should go to `accountsservice.nix`. There might be other extensions than lightdm.

Reply:
Right. Done.